### PR TITLE
Unify page styling

### PR DESCRIPTION
**Pull request by fhildeb**
This PR:

- fixes ESLink build errors on `'` characters
- fixes broken `columns`/`column` styles so all pages have equal container widths
- adds rounded corners to `code` + `pre` elements to align with overall look

### CODE REVIEW

**Review by Esanim — APPROVED** [review]
All good for me.

File: components/Encode/components/EncodeSetData.tsx [hunk 1]
```diff
@@ -124,8 +124,8 @@ const EncodeSetData: React.FC<Props> = ({ web3, isBatch }) => {
     <>
       {createInputs(keyValuePairs)}
       {isBatch ? (
-        <div className="columns">
-          <div className="column">
+        <div>
+          <div className="my-4 mr-4">
```
Comment (Hugoo) [inline components/Encode/components/EncodeSetData.tsx:128]: I'm not certain this mr-4 is responsive. I preferred the previous way with columns.

File: components/KeyManagerPermissions/KeyManagerPermissions.tsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ const KeyManagerPermissions: React.FC = () => {
       <h2 className="title is-2">Permissions</h2>
       <article className="message is-info">
         <div className="message-body">
-          Encode and decode{' '}
+          This tool will encode and decode{' '}
```
Comment (Hugoo) [inline components/KeyManagerPermissions/KeyManagerPermissions.tsx:44]: No need for "this tool will" you can leave it as it was before.

**Comment by Hugoo** [comment]
What was broken with columns?

Maybe you can fix the columns rather than removing them?

https://bulma.io/documentation/columns/basics/

File: components/Encode/components/EncodeSetData.tsx (hunk 1 above)
Comment (fhildeb) [reply components/Encode/components/EncodeSetData.tsx:128]: Columns class is overwriting the left offset of the element of the container. Thats why i removed it. All margin classes are responsive (including `mr`) as they use `rem`

**Pull request merged**